research/management/commands/orcid-meta-import.py

```diff
@@ -0,0 +1,300 @@
+from django.core.management.base import BaseCommand, CommandError
+from research.models import ResearchWork, Researcher, ResearcherEducation
+from teledata.models import Staff
+
+from django.core.management.base import BaseCommand, CommandError
+from research.models import Researcher
+from teledata.models import Staff
```

Comment:
Researcher and Staff imports on line 6 and 7 can be removed (already imported on lines 2 and 3)